content/docs/user-guide/platforms/linux/_index.md

```diff
@@ -5,64 +5,118 @@ description: An overview of Open 3D Engine support for development on Linux.
-O3DE will also run on Ubuntu 20.04 LTS, but requires the installation of [libffi6](http://mirrors.kernel.org/ubuntu/pool/main/libf/libffi/libffi6_3.2.1-8_amd64.deb).
+When generating projects, a project path is required. This can be an existing O3DE project, or a new one generated with the instructions provided through the [Projects](docs/user-guide/project-config) page. The instructions below will assume that $O3DE_PROJECT will refer to the absolute path of the project to base the build on. 
 {{< /note >}}
 
-O3DE also requires that it have its own Python runtime installed. Before running any O3DE scripts or performing a CMake operation, install the O3DE Python runtime package.
+
+### Multiple Config Builds
+The following command will generate a build folder under the root of the project folder $O3DE_PROJECT called build/linux, using Ninja as the build generator and clang-12 as the compiler.
+
+```shell
+
+cd $O3DE_PROJECT
+
+$O3DE_ENGINE/scripts/o3de.sh register --project-path .
+
+cmake -B build/linux -S . -G"Ninja Multi-Config" -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12 
+
+```
+
+### Single Config Builds
+
+If you do not need to generate multi-config build folders, you can instead specify "Unix Makefiles" instead, but will need to specify a configuration the configuration at the time of project generation. ($BUILD_CONFIG below can be either debug, profile, or release)
+
+```shell
+
+cd $O3DE_PROJECT
+
+$O3DE_ENGINE/scripts/o3de.sh register --project-path .
+
+cmake -B build/linux -S . -G"Unix Makefiles" -DCMAKE_C_COMPILER=clang-12 -DCMAKE_CXX_COMPILER=clang++-12 -DCMAKE_BUILD_TYPE=$BUILD_CONFIG
+
+```
+
+To build the project, you can specify a specific target (like Editor for the example below)
+
+```shell
+
+cd $O3DE_PROJECT
+
+cmake --build build/linux --target=Editor
+```
+
+
+## Building from command line
+
+Once the build folder is generated, building from command line at this point is the same as the other platforms. 
+
+### Single Config Builds
+
+For projects that were generated using the Unix Makefile generator, the configuration will already be specified during the project generation and you can only build for that configuration.
 
 ```shell
-cd <o3de-source-dir>
-./python/get_python.sh
+
+cd $O3DE_PROJECT
+
+cmake --build build/linux 
 ```
 
-## Configuration and build
 
-O3DE supports using the "Unix Makefiles" generator with CMake.
+### Multiple Config Builds
 
-## Install O3DE from source
+For projects that were generated using the Ninja Multi-Config generator will require that the build configuration be part of the build command.
 
 ```shell
-cd <o3de-source-directory>
-mkdir -p build/linux-dedicated
-cmake -B build/linux-dedicated -S . \
-    -DLY_3RDPARTY_PATH=<absolute-path-to-packages> \
-    -DLY_UNITY_BUILD=ON \
-    -DCMAKE_INSTALL_PREFIX=<install-location> \
-cmake --build build/linux-dedicated --config profile --target install
+
+cd $O3DE_PROJECT
+
+cmake --build build/linux --config profile
```

Comment:
Same question as above about needing a `--target` argument here...

Reply:
By default, it builds all targets (include project manager), which probably isn't applicable for new projects, but is applicable if they are building O3DE straight from source code only.

Reply:
Scoped the targets